modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerPartitionReconciliationExtendedTest.java

```diff
@@ -133,8 +133,15 @@ public void testPartitionReconciliationCancel() throws Exception {
     @Test
     @WithSystemProperty(key = "RECONCILIATION_WORK_PROGRESS_PRINT_INTERVAL", value = "0")
     public void testProgressLogPrinted() throws Exception {
-        LogListener lsnr = LogListener.matches(s -> s.startsWith("Partition reconciliation task [sesId=")).atLeast(1).build();
-        log.registerListener(lsnr);
+        LogListener lsnr1 = LogListener.matches(s -> s.startsWith("Partition reconciliation status [sesId=")).atLeast(1).build();
```

Comment:
I'd also check the fact that `processedPrimaryKeys` and `processedBackupKeys` were actually updated within multiple runs of printStatistics. And that last corresponding log record contains expected amount of primary and backup keys.